data/tosca/yorc-slurm-types.yml

```diff
@@ -47,6 +47,10 @@ node_types:
         type: string
         required: false
         description: Specify a name for the job allocation. The specified name will appear along with the job id.
+      account:
+        type: string
+        description: >
+          Charge resources used by this allocation to specified account. May be mandatory according to configuration.
```

Comment:
Could use the same description in `yorc.nodes.slurm.Job` property `account` (add that the property may be mandatory according to configuration)

Reply:
Yes